src/main/java/net/revelc/code/formatter/xml/internal/FormattingPreferences.java

```diff
@@ -13,7 +13,7 @@
  * 	   IBM Corporation - bug fixes
  * 	   Jose Montoya - Modified implementation outside Eclipse Platform
  *******************************************************************************/
-package net.revelc.code.formatter.xml;
```

Comment:
How about `lib` instead of `internal`?

Reply:
Sure, done. I have no particular preferences, just the first thing that came to mind.